### PR TITLE
Fixing flake8 errors

### PR DESCRIPTION
**Pull request by mpope9**
@avadhanij  I realize this is a bit long, but I ran flake8 and fixed all but 1 error. Can you take a look and see if the style is better?

### CODE REVIEW

**Comment by avadhanij** [comment]
> Damn, flake8 reports 602 violations

Haha, yeah, and they show up big time if you are using an IDE like PyCharm or VSCode. 

This is very impressive. I checked out the branch locally, and it looks good.

One thing I will recommend is to avoid the flake8 default of 80 character line length. Personally, I find it very limiting. In an era of wide screen and even ultra wide screen curved monitors, this just doesn't make a lot of sense. 

Imo, 120 characters is better. Can stretch it further too, but it gets a little too long on a smaller screen like a 13 inch device.

To instruct flake8 to use some of your preferences instead, you need to add a `.flake8` file into the repo. Here is something you can start with -

```
[flake8]
count=True
filename=*.py
exclude =
  .git,
  .conf,
  __pycache__,
  build,
  dist,
  *.egg-info,
  *.ini,
  *.conf,
  *.flake8,
  *.yaml
max-line-length=120
```

That last line will tell flake8 to complain only if the length crosses 120 chars.

**Review by avadhanij — APPROVED** [review]
Can you merge it? I can add the flake8 file.

I want to try out some stuff, and since the changes here have touched every file extensively, it will result in merge conflicts.

**Comment by mpope9** [comment]
@avadhanij Sure will merge. I haven't given it a run, but since flake8 doesn't complain I'll assume it compiles file.

File: stats/nba_sql.py [hunk 1]
```diff
@@ -193,20 +235,27 @@ def do_create_schema(object_list):
     for obj in object_list:
         obj.create_ddl()
 
-def populate_base_tables(seasons, request_gap, team_requester, player_requester, event_message_type_builder):
+
+def populate_base_tables(
```
Comment (avadhanij) [inline stats/nba_sql.py:239]: Btw, is this a convention, you want to adopt? i.e, every parameter on a different line? If you set `max-line-length` to 120 chars, then you wouldn't see flake8 complain.

Comment (mpope9) [reply stats/nba_sql.py:239]: I'd prefer not to. I think 120 is good, I will go back and make these less insane. Was just trying to get it to a baseline.

**Pull request merged**